stage_upgrade.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+
+# This script will stage the upgrade files for casper network from 1.4.3 to 1.4.4
```

Comment:
this says 1.4.3 to 1.4.4 but should it be 1.4.4 to 1.4.5 ?

Reply:
Yep, corrected.